src/main/resources/db/migration/common/V1_0_40__add-kpi-migration-columns.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE kpi
+    ADD COLUMN IF NOT EXISTS system_source VARCHAR(10);
+
+UPDATE kpi
+    SET system_source = 'UI'
+    WHERE system_source is null;
+
+ALTER TABLE kpi
+    ADD COLUMN IF NOT EXISTS original_id VARCHAR(36);
```

Comment:
Probably useful to name it gateway_id instead of original_id?

Reply:
true, I'll change it.